homeassistant/components/automation/config.py

```diff
@@ -79,8 +79,21 @@ async def async_validate_config_item(hass, config, full_config=None):
     return config
 
 
+class AutomationConfig(dict):
+    """Dummy class to allow adding attributes."""
+
+    raw_config = None
+
+
 async def _try_async_validate_config_item(hass, config, full_config=None):
     """Validate config item."""
+    raw_config = None
+    try:
+        raw_config = dict(config)
+    except ValueError:
+        # Invalid config
+        pass
```

Comment:
Shouldn't we raise?

Reply:
I didn't want to interfere with the config validation which is handled just below.